### DISCUSSION TITLE
Can dokku be used with kubernetes?

### DISCUSSION THREAD

**Discussion by mjr** [dokku/dokku#4666] [Q&A] [answered]
If yes, do you have any documentation about it?

**Comment by josegonzalez** [comment, marked as answer]
There is a kubernetes plugin here: https://github.com/dokku/dokku-scheduler-kubernetes

**Reply by mjr** [reply]
Sorry, but i don't know how this would work, do you have any documentation/tutorial on how to do/configure this?

**Reply by josegonzalez** [reply]
The readme contains setup instructions: https://github.com/dokku/dokku-scheduler-kubernetes#usage
